crm_phonecall/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Crm Phonecalls
+==============
+
+* This module allows to manage phone calls in order to analyze them.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to *Sales > Phone Calls > Logged Calls > Create*.
```

Comment:
Use `#.` for a numbered list instead a bullet list. Check that indentation of the second line is correct (with the new delimiter, it will, as it requires 3 space, but now, it should be 2, as the text starts in that position).